### PR TITLE
Lovelace: add confirmation action

### PR DESCRIPTION
**Pull request by iantrich**
**Description:**

https://github.com/home-assistant/home-assistant-polymer/pull/4006

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by iantrich** [comment]
Is there a better way to do this? Repeating the same lines throughout all these files seems prone to failure and makes maintenance a chore.

File: source/_includes/asides/lovelace_navigation.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <section class="aside-module grid__item one-whole lap-one-half">
   <!-- {% include edit_github.html %} -->
   {% assign cards = site.lovelace | sort_natural: 'title' %}
+  {% assign cards = site.lovelace.badges | sort_natural: 'title' %}
```
Comment (frenck) [inline source/_includes/asides/lovelace_navigation.html]: This now overwrites the line above...
Resulting in losing all the Lovelace cards? 😒

File: source/_lovelace/entity-button.markdown [hunk 2]
```diff
@@ -110,6 +115,11 @@ hold_action:
       description: "Service data to include (e.g. `entity_id: media_player.bedroom`) when `action` defined as `call-service`"
       type: string
       default: none
+    confirmation:
+      required: false
+      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
+      type: [boolean, map]
+      default: "false"
```
Comment (frenck) [inline source/_lovelace/entity-button.markdown:122]: You don't have to quote booleans ;)

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_includes/asides/lovelace_navigation.html (hunk 1 above)
Comment (iantrich) [reply source/_includes/asides/lovelace_navigation.html]: Opps, didn't even mean to have this 🤦‍♂

File: source/_lovelace/entity-button.markdown (hunk 2 above)
Comment (iantrich) [reply source/_lovelace/entity-button.markdown:122]: All other defaults in this file had quotes, so I followed the trend 🤷‍♂

Comment (frenck) [reply source/_lovelace/entity-button.markdown:122]: 👍 I know! It's weird! 😉

**Review by frenck — APPROVED** [review]

**Pull request merged**